tools/RegenerateExpectedClasses.ps1

```diff
@@ -1,8 +1,8 @@
 Write-Output ">>>>>>>>> Generating CSharp clients"
```

Comment:
I think the intention is to run this from the top level, with 'powershell tools\RegenerateExpectedClasses.ps1'